### PR TITLE
Clean Architecture Service Template Added

### PR DESCRIPTION
**Pull request by ivanpaulovich**
Clean Architecture service template for your Microservice with DDD, TDD and SOLID using .NET Core 2.0. The components are independent and testable, the architecture is evolutionary in multiple dimensions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -681,6 +681,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
  [Mobile Apps](https://github.com/Microsoft/BikeSharing360_MobileApps), [Backend Services](https://github.com/Microsoft/BikeSharing360_BackendServices), [Websites](https://github.com/Microsoft/BikeSharing360_Websites), [Single Container Apps](https://github.com/Microsoft/BikeSharing360_SingleContainer), [Multi Container Apps](https://github.com/Microsoft/BikeSharing360_MultiContainer), [Cognitive Services Kiosk App](https://github.com/Microsoft/BikeSharing360_CognitiveServicesKioskApp),
  [Azure Bot App](https://github.com/Microsoft/BikeSharing360_BotApps).
 * [cloudscribe](https://github.com/cloudscribe/cloudscribe) - ASP.NET Core Multi-tenant web application foundation.
+* [clean-architecture-manga](https://github.com/ivanpaulovich/clean-architecture-manga) - Clean Architecture service template for your Microservice with DDD, TDD and SOLID using .NET Core 2.0. The components are independent and testable, the architecture is evolutionary in multiple dimensions.
```
Comment (ngohungphuc) [inline README.md]: Is this application template if so please move to that section. Thank you

**Comment by Jogai** [comment]
Really aweseome project. Question, is it really a template if you cant do 'dotnet new manga' for example? Or trough visual studio project templates?

**Comment by ivanpaulovich** [comment]
@Jogai you can use simple as:

````
dotnet new -i Paulovich.Manga
dotnet new manga
````
It will generate an application with default switches. Try `dotnet new manga --help` for the complete options.

**Comment by ngohungphuc** [comment]
Thank for your PR Ivan

**Pull request merged**